evalutils/stats.py

```diff
@@ -8,13 +8,19 @@
 from scipy.ndimage.morphology import binary_erosion, generate_binary_structure
 
 
+VOXELSPACING_TYPE = Optional[
+    Union[Tuple[Union[float, int], ...], List[Union[float, int]], float, int]
+]
+
+
 def distance_transform_edt_float32(  # noqa: C901
-    input,
-    sampling=None,
-    return_distances=True,
-    return_indices=False,
-    distances=None,
-    indices=None,
+    input: ndarray,
+    sampling: VOXELSPACING_TYPE = None,
+    return_distances: bool = True,
+    return_indices: bool = False,
+    distances: Optional[ndarray] = None,
+    indices: Optional[ndarray] = None,
+    gc_byte_threshold: int = 100000000,
```

Comment:
Can you remove this extra argument? The function signature is intentionally the same as the one used in scipy.

Reply:
Hmmm yes, I can see how that's desirable. I introduced the argument only for testing the garbage collection conditions. 
If it is removed it cannot be tested in a reasonable time which in turn will reduce the coverage. I'll try to look for an alternative solution here.

Reply:
You could declare this on the module level and then modify/patch that, but, I don't see testing those lines as that important.

Reply:
Ok, I reverted the method to its old form, so the `gc_byte_threshold` param is gone.